.ci/integrationTestSelector.groovy

```diff
@@ -219,12 +219,13 @@ def wrappingup(Map params = [:]){
   dir("${BASE_DIR}"){
     sh("./scripts/docker-get-logs.sh '${env.NAME}'|| echo 0")
     sh('make stop-env || echo 0')
+    def testResultsPattern = '**/tests/results/*-junit*.xml'
     archiveArtifacts(
         allowEmptyArchive: true,
-        artifacts: 'docker-info/**,**/tests/results/data-*.json,,**/tests/results/packetbeat-*.json',
+        artifacts: "docker-info/**,**/tests/results/data-*.json,,**/tests/results/packetbeat-*.json,${testResultsPattern}",
         defaultExcludes: false)
     if (isJunit) {
-      junit(allowEmptyResults: true, keepLongStdio: true, testResults: "**/tests/results/*-junit*.xml")
+      junit(allowEmptyResults: true, keepLongStdio: true, testResults: testResultsPattern)
```

Comment:
The step accepts variables, so no need to interpolate its value, does it?

Reply:
It supports variables yep :)